src/_layouts/base.html

```diff
@@ -9,7 +9,7 @@
     <link rel="icon" href="/assets/images/shared/brand/flutter/icon/64.png" />
     <link rel="apple-touch-icon" href="/assets/images/shared/brand/flutter/logo/flutter-logomark-320px.png" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <meta name="theme-color" content="#ffffff" />
+    <meta name="theme-color" content="#17a2b8" />
```

Comment:
**issue:** I believe this should be the same color as the link color:
```suggestion
    <meta name="theme-color" content="#0175C2" />
```

If it's the same as the background of callouts, that would be `#2372a3`.

@kevmoo , any thoughts?

Reply:
Maybe? I'd love to see a screenshot of the output on mobile for context...